lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -199,6 +208,20 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private String getVCStringFromCredentialJwt(String riskAssessmentCredential)
```

Comment:
Does this method do anything, aside from parse and re-serialise the JWT? I think we can probably just pass the VC string directly, since we should have already validated it when received from TICF.

Reply:
done